runtime/src/main/java/org/corfudb/runtime/clients/IClient.java

```diff
@@ -20,6 +21,14 @@
      */
     void setRouter(IClientRouter router);
 
+    /**
```

Comment:
It seems like this type should be an abstract class instead of an interface. Might change it later.